test/any_unique_test.cpp

```diff
@@ -26,6 +26,7 @@
 #include <gtest/gtest.h>
 
 namespace {
+#if __cpp_rtti
 inline constexpr struct get_typeid_cpo {
   using type_erased_signature_t =
       unifex::type_index(const unifex::this_&) noexcept;
```

Comment:
I introduced this type in PR #154; its definition is _supposed_ to depend on whether `__cpp_rtti` is defined or not.  I wouldn't expect you'd need to compile out this CPO.  If you do, maybe there's a deeper change that's needed?

Reply:
Ah, you're right. I got a little overenthusiastic. Will remove.